CodeCoverageQualityGate/CodeCoverageQualityService.ts

```diff
@@ -288,8 +278,7 @@ export class CodeCoverageQualityService {
                     referenceBuildInfo.warningMessage = `no Parasoft coverage results were found in any of the previous successful builds in pipeline '${pipelineName}'`;
                     return referenceBuildInfo;
                 case DefaultBuildReportResultsStatus.NO_PREVIOUS_BUILD_WAS_FOUND:
-                    referenceBuildInfo.warningMessage = `No previous build was found in pipeline '${pipelineName}'`;
```

Comment:
Please double check with @rliu-parasoft for the reason why she set `isDebugMessage` flag and capitalized the first letter of this message here.